lib/SILOptimizer/Mandatory/AddressLowering.cpp

```diff
@@ -3101,6 +3101,17 @@ class UseRewriter : SILInstructionVisitor<UseRewriter> {
     llvm::report_fatal_error("Unimplemented SelectValue use.");
   }
 
+  void visitStoreBorrowInst(StoreBorrowInst *sbi) {
+    auto addr = addrMat.materializeAddress(use->get());
+    for (auto *use : sbi->getUses()) {
+      if (auto *ebi = dyn_cast<EndBorrowInst>(use->getUser())) {
+        pass.deleter.forceDelete(ebi);
+      }
```

Comment:
can this called use iterator invalidation when there are multiple end_borrows ?

Reply:
Thanks!  Fixed.